src/pages/TicketList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import HotelCarousel from '../components/ticketList/HotelCarousel';
+import Ticket from '../components/ticketList/Ticket';
+
+const TicketList = () => {
+  return (
+    <>
+      <HotelCarousel></HotelCarousel>
+      <Ticket></Ticket>
```

Comment:
Children이 필요 없는 컴포넌트는 self-closing tag로 써주면 좋을거같아요!!
```suggestion
      <HotelCarousel />
      <Ticket />
```

Reply:
헉 넹 ! 반영하겠슴다